src/components/Visualisers/Map.tsx

```diff
@@ -0,0 +1,191 @@
+  flowRate?: string,
+}
+
+const Map: FC<{
+  network: Network;
+  accountAddress: string;
+}> = ({network, accountAddress}): ReactElement => {
+
+  const initialNodes: node[] = [
+    {
+      id: accountAddress,
+      position: { x: 500, y: 300 },
+      data: {
+        label:
+        <UserBlock network={network!} address={accountAddress} />
+      }
+    },
+  ];
+
+  const initialEdges: edge[] = [{ id: 'e1-2', source: '1', target: '2' }];
+
+  const [nodes, setNodes, onNodesChange] = useNodesState(initialNodes);
+  const [edges, setEdges, onEdgesChange] = useEdgesState(initialEdges);
+
+  const onConnect = useCallback((params: any) => setEdges((eds) => addEdge(params, eds)), [setEdges]);
+
+  //query incoming and outgoing streams on user address
+  const outgoingStreams = sfSubgraph.useStreamsQuery({
+    chainId: network.chainId,
+    filter: {
+      sender: accountAddress
+    },
+    pagination: {
+      take: Infinity
+    }
+  });
+
+  const incomingStreams = sfSubgraph.useStreamsQuery({
+    chainId: network.chainId,
+    filter: {
+      receiver: accountAddress
+    },
+    pagination: {
+      take: Infinity
+    }
+  });
+
+  //Filter out 0 current flow rate streams
+  const OutgoingStreams: any = useMemo(() => (outgoingStreams.data?.data.filter((row) => row.currentFlowRate !== '0') || []), [outgoingStreams.data]);
+  const IncomingStreams: any = useMemo(() => (incomingStreams.data?.data.filter((row) => row.currentFlowRate !== '0') || []), [incomingStreams.data]);
+
+
+  //Set up edges and nodes with streams
+  useEffect(() => {
+    if (!IncomingStreams || !OutgoingStreams) {
+      return;
+    }
+
+    let incomingNodeList: node[] = [initialNodes[0]];
+    let outgoingNodeList: node[] = [initialNodes[0]];
+    let edgeList: edge[] = [];
+    let outgoingEdgeList: edge[] = [];
+
+    IncomingStreams.map((row: any, i: number) => {
+      if (row.currentFlowRate === '0') {
+        return
+      }
+      let ethersFlowRate = ethers.utils.formatEther(row.currentFlowRate);
+      //Convert to humanized monthly amount
+      let humanizedFlowRate = +ethersFlowRate * 3600 * 24 * 30;
+      //Create a node object with relevant data
+      let node = {
+        id: `${row.sender}-${i}`,
+        position: {x: i*300, y: 100},
+        data: {
+          label: <UserBlock network={network} address={row.sender}/>
+        },
+        flowRate: `${humanizedFlowRate.toFixed(2)}/Mo  ${row.tokenSymbol}`,
+      }
+      incomingNodeList.push(node);
+    })
+
+    //Make these come out from TOP of Node
+    incomingNodeList.map((node: node, i: number) => {
+      if (i === incomingNodeList.length) {
+        return;
+      }
+      let edge = {
+        id: `e${node.id}-${accountAddress}`,
+        label: node.flowRate,
+        source: node.id,
+        target: accountAddress,
+        animated: true
+      }
+      edgeList.push(edge);
+    })
+
+    OutgoingStreams.map((row: any, i: number) => {
+      if (row.currentFlowRate === '0') {
```

Comment:
This check is useless because zero flow rate rows should be already filtered out.